packages/shared/src/lib/featureValues.ts

```diff
@@ -22,13 +22,11 @@ export enum OnboardingFiltersLayout {
   List = 'list',
 }
 
-export enum ArticleOnboardingVersion {
+export enum ScrollOnboardingVersion {
   V1 = 'v1',
   V2 = 'v2',
-  V3 = 'v3',
 }
 
-export enum ScrollOnboardingVersion {
-  V1 = 'v1',
-  V2 = 'v2',
+export enum ExperimentWinner {
+  ArticleOnboarding = 'v3',
 }
```

Comment:
Like this approach, but perhaps we should always aim to fully cleanup.
In which case is this value still used?

Reply:
NVM, see next comment 🙏